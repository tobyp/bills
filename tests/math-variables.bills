#!./bills
%set x 5
%set y 7
%set z 11
A	B.	x+y*2+z/3
