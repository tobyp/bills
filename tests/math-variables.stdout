todo: B pays 22.666666 to A
